scripts/ansible/roles/linux/common/vars/ubuntu/bionic.yml

```diff
@@ -2,5 +2,4 @@
 # Licensed under the MIT License.
 
 ---
-llvm_apt_repository: "deb http://apt.llvm.org/{{ ansible_distribution_release }} llvm-toolchain-{{ ansible_distribution_release }}-7 main"
-llvm_apt_repository_10: "deb http://apt.llvm.org/{{ ansible_distribution_release }} llvm-toolchain-{{ ansible_distribution_release }}-10 main"
+llvm_apt_repository: "deb http://apt.llvm.org/{{ ansible_distribution_release }} llvm-toolchain-{{ ansible_distribution_release }}-10 main"
```

Comment:
Could you also make the same change for Ubuntu 20.04?
https://github.com/openenclave/openenclave/blob/master/scripts/ansible/roles/linux/common/vars/ubuntu/focal.yml#L6

Reply:
As we discussed offline @manojrupireddy , we want keep the `llvm_apt_repository` variable name, but use `llvm-toolchain-<version>-10`.